acc_test/resource_vpc_peering_test.go

```diff
@@ -2,9 +2,9 @@ package test
 
 import (
 	"fmt"
-//	"io/ioutil"
-//	"os"
-//	"testing"
+	//	"io/ioutil"
```

Comment:
**[QUESTION]** Was this spacing intended?

Reply:
It appears after merging master branch, but in my local I don't have these changes. Should be fine.